Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1945,3 +1928,34 @@ void MainWindow::resetHeader()
 }
 
 
+void MainWindow::reset_default_loaders()
+{
+  default_plugin_selection.clear();
+
+  const char* prop_name = "Menu modified by MainWindow.";
+  QMenu* menu = ui->menuFile;
+  if(!menu)
+    return;
+  bool menuChanged = menu->property(prop_name).toBool();
+  if(!menuChanged) {
+    menu->setProperty(prop_name, true);
+  }
+  QList<QAction*> menuActions = menu->actions();
+  menu->removeAction(actionResetDefaultLoaders);
+}
+
+void MainWindow::insertActionBefore(QMenu* menu, QAction* actionToInsert)
+{
+  const char* prop_name = "Menu modified by MainWindow.";
+  if(menu)
+  {
+    bool menuChanged = menu->property(prop_name).toBool();
+    if(!menuChanged) {
+      menu->setProperty(prop_name, true);
+    }
```

Comment:
I am pretty sure you can remove `prop_name`, `menuChanged` amd the `setProperty(prop_name, true)`. That property is always set to `true`, and its value does not change the behavior of the functions.